gammapy/irf/tests/test_energy_dispersion.py

```diff
@@ -68,21 +73,29 @@ def test_EnergyDispersion2D():
     desired = edisp.dispersion[off_node, m_node, e_node]
     assert_allclose(actual, desired, rtol=1e-06)
 
-    # Check that values between node make sense
-    # TODO: THINK ABOUT WHAT MAKES SENSE
-    energy2 = edisp.energy[e_node + 1]
-    upper = edisp.evaluate(offset, energy, migra)
-    lower = edisp.evaluate(offset, energy2, migra)
-    e_val = (energy + energy2) / 2
-    actual = edisp.evaluate(offset, e_val, migra)
-    # assert_equal(lower > actual and actual > upper, True)
+    # Check evaluation at all nodes
+    actual = edisp.evaluate().shape
+    desired = (len(edisp.offset), len(edisp.migra), len(edisp.energy))
+    assert_equal(actual, desired)
+
+    # Get response
+    e_reco = EnergyBounds.equal_log_spacing(1 * u.GeV, 100 * u.TeV, 100)
+    response = edisp.get_response('1 deg', '1 TeV', e_reco)
+    actual = len(response)
+    desired = e_reco.nbins
+    assert_equal(actual, desired)
+
+    # Check normalization
+    actual = np.sum(response)
+    desired = 1
+    assert_allclose(actual, desired, rtol=1e-1)
```

Comment:
@cdeil : Do we need a better agreement with 1 here?
I can't judge if this is a binning effect or something else. I could force this to be 1 by renormalizing of course.